tests/RobotFramework/tasks.py

```diff
@@ -323,13 +352,19 @@ def test(
 
     if adapter == ADAPTER_DOCKER:
         container_cli = detect_container_cli()
+        network_name = "inttest-network"
 
         # create docker network that is used by each container
         # Create before launching tests otherwise there will be a race condition
         # which causes multiple networks with the same name to be created.
+        log.info("Creating a container network")
         c.run(
-            f"command -v {container_cli} &>/dev/null && ({container_cli} network create inttest-network --driver bridge || true) || true"
+            f"command -v {container_cli} &>/dev/null && ({container_cli} network create {network_name} --driver bridge || true) || true"
         )
+
+        # Required because of docker race condition which leads to duplicate networks
```

Comment:
When we say remove duplicate networks, are all the container networks the same type?

Reply:
The same container network is used by all tests, as only one network is needed, and container just connect and disconnect from it when the containers are created and destroyed.